### ISSUE TITLE
Pyright stopped recognizing types for numpy and OpenCV

### ISSUE THREAD

**Issue by iansan5653** [microsoft/pyright#295]
**Describe the bug**
Until now, Pyright would automatically recognize members of numpy and OpenCV in my project. For example, `np.ndarray` was a valid type. Now I get the following error:

> 'array' is not a known member of module

**Screenshots or Code**
This file replicates the problem for me:
```py
import numpy as np


def ex(x: np.ndarray):
    ...
```
![image](https://user-images.githubusercontent.com/2294248/67229502-6cdfcc80-f409-11e9-83c4-708d39c51db9.png)

**VS Code extension or command-line**
I'm running version 1.0.78 of the extension in version 1.40.0 of VSCode with Python 3.7.4 64-bit on Windows 10.

**Additional context**
`settings.json` for the workspace:
```json
{
  "editor.tabSize": 4,
  "python.testing.promptToConfigure": false,
  "python.testing.pytestEnabled": false,
  "python.testing.unittestEnabled": false,
  "python.testing.nosetestsEnabled": false,
  "python.pythonPath": "C:\\Users\\Ian Sanders\\AppData\\Local\\Programs\\Python\\Python37\\python.exe"
}
```
`pyrightconfig.json`:
```json
{
  "strictListInference": true,
  "strictDictionaryInference": true,
  "strictParameterNoneValue": true,
  "enableTypeIgnoreComments": true,
  "reportTypeshedErrors": true,
  "reportMissingImports": true,
  "reportMissingTypeStubs": false,
  "reportImportCycles": true,
  "reportUnusedImport": true,
  "reportUnusedClass": true,
  "reportUnusedFunction": true,
  "reportUnusedVariable": true,
  "reportOptionalSubscript": true,
  "reportOptionalMemberAccess": true,
  "reportOptionalCall": true,
  "reportOptionalIterable": true,
  "reportOptionalContextManager": true,
  "reportOptionalOperand": true,
  "reportUntypedFunctionDecorator": true,
  "reportUntypedClassDecorator": true,
  "reportUntypedBaseClass": true,
  "reportUntypedNamedTuple": true,
  "reportPrivateUsage": true,
  "reportConstantRedefinition": true,
  "reportIncompatibleMethodOverride": true,
  "reportInvalidStringEscapeSequence": true,
  "reportUnknownParameterType": true,
  "reportUnknownVariableType": false,
  "reportUnknownMemberType": false,
  "reportCallInDefaultInitializer": true,
  "reportUnnecessaryIsInstance": true,
  "reportUnnecessaryCast": true,
  "executionEnvironments": [
    {
      "root": "code"
    }
  ]
}
```
File structure:
* workspace folder
  * code
    * my_file.py

It's probably irrelevant but for completeness, my user `settings.json`:
<details>
<summary>`settings.json`</summary>
```json
{
    "files.autoSave": "afterDelay",
    "git.confirmSync": false,
    "git.autofetch": true,
    "liveServer.settings.donotVerifyTags": true,
    "liveServer.settings.donotShowInfoMsg": true,
    "git.enableSmartCommit": true,
    "explorer.confirmDragAndDrop": false,
    "php.validate.enable": false,
    "material-icon-theme.showUpdateMessage": false,
    "editor.renderWhitespace": "none",
    "editor.renderControlCharacters": true,
    "editor.fontFamily": "Fira Code",
    "editor.rulers": [80],
    "explorer.confirmDelete": false,
    "window.menuBarVisibility": "default",
    "markdown.extension.toc.githubCompatibility": true,
    "markdown.extension.toc.levels": "2..6",
    "liveServer.settings.port": 4401,
    "window.zoomLevel": 0,
    "editor.colorDecorators": true,
    "window.titleBarStyle": "custom",
    "editor.wordWrap": "off",
    "workbench.settings.settingsSearchTocBehavior": "filter",
    "todohighlight.defaultStyle": {
      "borderRadius": "2px",
      "fontWeight": "bold",
      "color": "rgb(33, 33, 33)"
    },
    "todohighlight.keywords": [
      {
        "text": "TODO:",
        "backgroundColor": "rgb(255, 210, 109)",
        "overviewRulerColor": "rgb(255, 210, 109)",
        "color": "rgb(33, 33, 33)"
      },
      {
        "text": "FIXME:",
        "backgroundColor": "rgb(240, 98, 146)",
        "overviewRulerColor": "rgb(240, 98, 146)",
        "color": "rgb(33, 33, 33)"
      },
      {
        "text": "MAYBE:",
        "backgroundColor": "rgb(66, 134, 244)",
        "overviewRulerColor": "rgb(66, 134, 244)"
      },
      {
        "text": "NOTE:",
        "backgroundColor": "#4f5359",
        "overviewRulerColor": "#4f5359",
        "color": "#fff",
        "fontWeight": "normal"
      },
      {
        "text": "INFO:",
        "backgroundColor": "rgba(0,0,0,0)",
        "overviewRulerColor": "rgba(0,0,0,0)",
        "color": "#fff",
        "fontWeight": "normal"
      },
      {
        "text": "DEBUG:",
        "backgroundColor": "rgba(0,0,0,0)",
        "overviewRulerColor": "rgba(0,0,0,0)",
        "color": "#bcbcbc",
        "fontWeight": "normal"
      },
      {
        "text": "WARNING:",
        "backgroundColor": "#e0af00",
        "overviewRulerColor": "#e0af00",
        "color": "#000",
        "fontWeight": "normal"
      },
      {
        "text": "ERROR:",
        "backgroundColor": "#db5700",
        "overviewRulerColor": "#db5700",
        "color": "#000",
        "fontWeight": "normal"
      },
      {
        "text": "CRITICAL:",
        "backgroundColor": "#bf1701",
        "overviewRulerColor": "#bf1701",
        "color": "#000",
        "fontWeight": "normal"
      }
    ],
    "markdown-pdf.headerTemplate": "",
    "files.associations": {
      "*.gs": "javascript"
    },
    "breadcrumbs.enabled": true,
    "editor.minimap.renderCharacters": false,
    "editor.minimap.maxColumn": 100,
    "sync.quietSync": false,
    "sync.removeExtensions": true,
    "sync.syncExtensions": true,
    "sync.autoDownload": false,
    "sync.autoUpload": false,
    "sync.forceDownload": false,
    "workbench.iconTheme": "material-icon-theme",
    "editor.tabCompletion": "on",
    "python.testing.pytestEnabled": true,
    "python.linting.flake8Enabled": true,
    "python.linting.flake8Args": [
      "--ignore=E501,W504,W503"
    ],
    "python.linting.banditArgs": ["--ignore=B101"],
    "python.testing.pytestArgs": ["--color=yes"],
    "gitlens.hovers.currentLine.over": "line",
    "gitlens.currentLine.enabled": false,
    "gitlens.heatmap.toggleMode": "window",
    "gitlens.views.repositories.location": "scm",
    "coverage-gutters.showGutterCoverage": false,
    "coverage-gutters.showLineCoverage": true,
    "coverage-gutters.partialHighlightDark": "rgba(121, 86, 10, 0.0)",
    "coverage-gutters.partialHighlightLight": "rgba(220, 213, 143, 0.2)",
    "coverage-gutters.noHighlightDark": "rgba(121, 31, 10, 0.2)",
    "coverage-gutters.noHighlightLight": "rgba(220, 143, 143, 0.2)",
    "coverage-gutters.highlightdark": "rgba(45, 121, 10, 0.0)",
    "coverage-gutters.highlightlight": "rgba(166, 220, 142, 0.0)",
    "coverage-gutters.showRulerCoverage": true,
    "coverage-gutters.ignoredPathGlobs": "**/{node_modules,venv,vendor,.tox,.pytest_cache,__pycache__,.git,.vscode}/**",
    "workbench.activityBar.visible": true,
    "python.formatting.provider": "yapf",
    "workbench.tree.indent": 6,
    "arduino.path": "C:\\Users\\isanders\\Documents\\Arduino IDE\\arduino-PR-beta1.9-BUILD-102",
    "arduino.defaultBaudRate": 9600,
    "editor.suggestSelection": "first",
    "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
    "javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
    "python.linting.pylintEnabled": false,
    "terminal.external.windowsExec": "cmd",
    "workbench.colorTheme": "Monokai",
    "[python]": {
      "editor.defaultFormatter": "ms-python.python"
    },
    "files.exclude": {
      "**/__pycache__/**": true,
      "**/.pytest_cache/**": true,
      "**/*.egg-info/**": true
    },
    "python.analysis.disabled": ["unresolved-import"],
    "vsintellicode.python.completionsEnabled": false,
    "editor.fontLigatures": true,
    "debug.console.fontFamily": "Fira Code",
    "terminal.integrated.rendererType": "dom",
    "markdown-table-formatter.markdownGrammarScopes": [
      "markdown",
      "javascript",
      "plaintext"
    ],
    "pythonTestExplorer.testFramework": "pytest",
    "[javascript]": {
      "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "editor.formatOnSave": true,
    "typescript.updateImportsOnFileMove.enabled": "always",
    "[html]": {
      "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[json]": {
      "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[typescript]": {
      "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "workbench.sideBar.location": "left",
    "[jsonc]": {
      "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "window.openFoldersInNewWindow": "on",
    "eslint.autoFixOnSave": true,
    "eslint.provideLintTask": true,
    "[markdown]": {
      "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "editor.tabSize": 2,
    "typescript.implementationsCodeLens.enabled": true,
    "coverage-gutters.coverageFileNames": [
      "lcov.info",
      "cov.xml",
      "jacoco.xml",
      "coverage/**/lcov.info",
      "coverage/**/cobertura-coverage.info"
    ],
    "eslint.alwaysShowStatus": true,
    "eslint.validate": [
      "javascript",
      "javascriptreact",
      "typescript",
      "typescriptreact",
      "js", "ts", "tsx", "jsx"
    ],
    "codemetrics.nodeconfiguration.ReturnStatement": 0,
    "codemetrics.nodeconfiguration.CaseClause": 0,
    "codemetrics.nodeconfiguration.DefaultClause": 0,
    "python.insidersChannel": "weekly",
    "prettier.proseWrap": "always",
    "python.testing.autoTestDiscoverOnSaveEnabled": false,
    "editor.formatOnType": true,
    "python.pythonPath": "C:\\Users\\Ian Sanders\\AppData\\Local\\Programs\\Python\\Python37\\python.exe"
  }
```
</details>

**Comment by erictraut** [comment]
Thanks for the bug report. I'm able to repro the problem. Appears to be a regression that I introduced in the last version. Apologies for that. I'll get a fix out by the end of the day.

**Comment by erictraut** [comment]
I just published 1.0.79, which addresses this problem. Thanks again for reporting it.

**Comment by iansan5653** [comment]
Confirmed, thanks!

**Comment by robertocarta** [comment]
I am still getting this problem with pyright 1.1.42

**Comment by erictraut** [comment]
@robertocarta, if you're seeing a problem, please open a new issue. It's almost definitely not the same as the one reported in this thread.
